skimage/io/tests/test_pil.py

```diff
@@ -203,6 +203,10 @@ def test_imsave_incorrect_dimension():
         with testing.raises(ValueError):
             with expected_warnings([fname + ' is a low contrast image']):
                 imsave(fname, np.zeros((2, 3, 2)))
+        # test that low contrast check is ignored
+        with testing.raises(ValueError):
```

Comment:
why does this raise valueerror?

Reply:
I don't know. I just amended the part that is related to the low contrast warning exception.

Reply:
It just doesn't like the fact that there are 2 channels in the last dimension of the array.
Not an ideal test, but I'm ok with how this looks now.

Thanks!

Reply:
I looked into it a little further, there isn't currently a place to put these "tests for all plugins". So this is as good of a place as any. I don't think PIL is going away any time soon.